src/app/containers/ArticleTimestamp/timeFormatTests/testUtils.js

```diff
@@ -0,0 +1,54 @@
+import * as moment from 'moment-timezone';
+import { timestampGenerator } from '../testHelpers';
+
+const timeFormat = 'D MMMM YYYY';
+
+export const timestampsFixtures = {
+  'exact date': {
+    time: 1539188425274,
+    isRelative: false,
+  },
+  'three hours ago': {
+    time: timestampGenerator({ hours: 3 }),
+    isRelative: true,
+  },
+  'five hours ago': {
+    time: timestampGenerator({ hours: 5 }),
+    isRelative: true,
+  },
+  'eleven hours ago': {
+    time: timestampGenerator({ hours: 11 }),
+    isRelative: true,
+  },
+  'twelve hours ago': {
+    time: timestampGenerator({ hours: 12 }),
+    isRelative: true,
+  },
+  'twenty four hours ago': {
+    time: timestampGenerator({
+      hours: 24,
+      seconds: 1,
+    }),
+    isRelative: true,
+  },
+  'two days ago': {
+    time: timestampGenerator({ days: 2 }),
+    isRelative: true,
+  },
+  'three days ago': {
+    time: timestampGenerator({ days: 3 }),
+    isRelative: true,
+  },
+};
```

Comment:
Nice examples however I have a few suggestions:

- Some of these are redundant, e.g. we do not currently show relative timestamps for >10h ago so there is no point having test cases for 'two days ago' (see https://github.com/bbc/simorgh/blob/latest/src/app/containers/ArticleTimestamp/helpers.js#L5-L8).
- I think we should have one test case that includes the time/timezone as well, like https://bbc.github.io/simorgh/?path=/story/containers-article-article-timestamp--lastpublished-firstpublished-and-firstpublished-today-and-10-hours-ago
- It would be good to have one case for minutes, e.g. "5 minutes ago"
- For both minutes and hours I think we should test singular and plural forms (i.e. *1 hour ago* vs *5 hours ago*) as these are important boundaries.